lib/fizzy/execute.cpp

```diff
@@ -687,7 +687,7 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             const auto br_table_size = read<uint32_t>(immediates);
             const auto arity = read<uint32_t>(immediates);
 
-            const auto br_table_idx = stack.pop();
+            const auto br_table_idx = static_cast<uint32_t>(stack.pop());
```

Comment:
This is more pedantic, than anything, because after validation, even if we do not clean up the stack, the only way to end up here is either using 32-bit ops or after `i32_wrap_i64`, which does the cleanup. I wonder if we miss something.

Could also fold this into being the first commit in #448.